C-Plus-Plus/cp/Array_Manipulation.cpp

```diff
@@ -0,0 +1,46 @@
+/*
+Problem link : https://www.hackerrank.com/challenges/crush/problem
+
+Problem description:
+Starting with a 1-indexed array of zeros and a list of operations, for each query add a value to each the array element between two given indices, inclusive. 
+Once all operations have been performed, return the maximum value in the array.
+*/
+
+#include <bits/stdc++.h>
+using namespace std;
+#define ll long long int
+
+int main()
+{
+    ll n, m, a, b, k;
+    cin >> n >> m;
+    vector<ll> prefix_sum(n + 1, 0);
+    for (ll queries = 1; queries <= m; queries++)
```

Comment:
Your main function should be the driver function. Make sure you write the logic of the code in a separate function and use main function just for calling the function.